src/shared/utilities.js

```diff
@@ -82,13 +82,27 @@ export const fetchPost = (url, body) => (
 );
 
 /**
- *
+ * Creates the paramTypes for a Twitter specific search term.
+ * @param array of strings representing search paramaterTypes
+ * @returns [{Object}] representing search terms with meta data
+ */
+export const createTwitterParamTypes = (selectedParamTypes) =>
+  ['author', 'hashtag', 'keyword', 'mention']
+    .map((paramType) => makeParamType(selectedParamTypes, paramType));

```

Comment:
@Winwardo I went a step further and mapped so didn't have to keep rewriting makeParamType.

Nice catch btw X)